src/scores/categorical/contingency_impl.py

```diff
@@ -570,7 +570,7 @@ def make_event_tables(
         return (forecast_events, observed_events)
 
     def make_contingency_manager(
-        self, forecast: FlexibleArrayType, observed: FlexibleArrayType, *, event_threshold=None, op_fn=operator.gt
+        self, forecast: FlexibleArrayType, observed: FlexibleArrayType, *, event_threshold=None, op_fn=None
```

Comment:
Could you do
```suggestion
        self, forecast: FlexibleArrayType, observed: FlexibleArrayType, *, event_threshold=None, op_fn=self.default_op_fn
```
to avoid the if statement that you introduced below on lines 587-589?

Reply:
I think that doesn't work in Python because it's not evaluated dynamically at that point. I can test it and give it a shot. I'd love to do that if it works. I'll test it but if it doesn't work as expected, I'll proceed to merge the current diff.

Reply:
Yep, 'self' is not defined at the time the method def is interpreted, so it can't be done that neatly

Reply:
side note: `dataclasses` seem to address this issue using a `__post_init__()` method. I've found them useful. Not necessary for this PR but just a design consideration potentially going down the track if we have more of these types of patterns.